### PR TITLE
Add a Github Action to clean up old images.

### PR DESCRIPTION
**Pull request by enasca**
This Github workflow executes once a day and cleans up images older than a month built by pull requests.

### CODE REVIEW

**Comment by enasca** [comment]
@hfrSchmidt As you can see from [this failed run](https://github.com/IKIM-Essen/EMCP-config/actions/runs/3445268502/jobs/5748785077), the default github token is insufficient in this case. You should have delete permissions on packages, so could you create an access token and add it to this workflow? Instructions at https://github.com/snok/container-retention-policy#token.

File: .github/workflows/image-cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          image-names: "*"
+          filter-tags: pr-*
+          cut-off: One month ago UTC
+          account-type: org
+          org-name: ${{ github.repository_owner }}
+          # The default Github token cannot be used here as deleting packages
+          # requires a personal access token with the appropriate permissions.
+          # See https://github.com/snok/container-retention-policy#token.
+          # token: ${{ secrets.PAT }}
```
Comment (hfrSchmidt) [inline .github/workflows/image-cleanup.yml:26]:
```suggestion
          token: ${{ secrets.PAT_HFRSCHMIDT }}
```

**Review by hfrSchmidt — CHANGES_REQUESTED** [review]

File: .github/workflows/image-cleanup.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Clean up old container images
+
+on:
```
Comment (hfrSchmidt) [inline .github/workflows/image-cleanup.yml:3]:
```suggestion
on:
  pull_request:
```

**Review by hfrSchmidt — COMMENTED** [review]
trigger the workflow for testing again

**Review by hfrSchmidt — APPROVED** [review]

**Pull request merged**